hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/store/contracts/precompile/BurnPrecompilesTest.java

```diff
@@ -24,6 +24,7 @@
 import static com.hederahashgraph.api.proto.java.HederaFunctionality.TokenBurn;
```

Comment:
When I look at the coverage of `BurnPrecompile` with these unit tests I see that coverage of the method `body` doesn't hit a line where it is building a v2 transaction - don't know why though - what's missing from these tests?

![image](https://github.com/hashgraph/hedera-services/assets/117694041/3f973e28-4007-4edf-8add-5d3909c39f58)